verification/truststore/truststore.go

```diff
@@ -0,0 +1,126 @@
+// Package truststore reads certificates in a trust store
+package truststore
+
+import (
+	"context"
+	"crypto/x509"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	corex509 "github.com/notaryproject/notation-core-go/x509"
+	"github.com/notaryproject/notation-go/dir"
+)
+
+// Type is an enum for trust store types supported such as
+// "ca" and "signingAuthority"
+type Type string
+
+const (
+	TypeCA               Type = "ca"
+	TypeSigningAuthority Type = "signingAuthority"
+)
+
+var (
+	Types = []Type{
+		TypeCA,
+		TypeSigningAuthority,
+	}
+)
+
+// X509TrustStore provide list and get behaviors for the trust store
+type X509TrustStore interface {
+	// List named stores under storeType
+	// List(ctx context.Context, storeType Type) ([]string, error)
```

Comment:
It seems this method is not implemented yet.

Reply:
I think we don't need this method. It is not used in the verification workflow.

Reply:
Will it be used in the `notation` CLI?